abstract/common-sotd.md

```diff
@@ -1 +1,3 @@
-During the Candidate Recommendation stage, the Working Group intends to demonstrate at least 2 implementations adopting this specification and making use of it, using a test suite.
\ No newline at end of file
+During the Candidate Recommendation stage, the Working Group will audit:
+* the [OpenTelemetry implementations of Baggage](https://opentelemetry.io/ecosystem/distributions/) to make sure they are following the W3C Baggage spec.
```

Comment:
The link here is for the third party distributions of OpenTelemetry - I don't think we have a plan to audit such third party distributions - my understanding is that we plan to audit only the official OpenTelemetry Baggage SDKs. Hence, should we update this to the below:

```suggestion
* the [OpenTelemetry implementations of Baggage](https://opentelemetry.io/docs/specs/otel/baggage/api/) to make sure they are following the W3C Baggage spec.
```